schemas/MobileScreen.json

```diff
@@ -44,9 +46,19 @@
         {"const": "Settings", "description": "The global settings screen shown in the app."},
         {"const": "SettingsSecurity", "description": "The global security settings screen."},
         {"const": "SettingsDefaultNotifications", "description": "The settings screen to change the default notification options."},
+        {"const": "SettingsGeneral", "description": "The settings screen with general profile settings."},
+        {"const": "SettingsNotifications", "description": "The notifications settings screen."},
+        {"const": "SettingsPreferences", "description": "The preferences screen (theme, language, editor preferences, etc."},
+        {"const": "SettingsVoiceVideo", "description": "The calls settings screen."},
+        {"const": "SettingsIgnoredUsers", "description": "The settings screen with list of the ignored users."},
+        {"const": "SettingsLabs", "description": "The experimental features settings screen,"},
+        {"const": "SettingsAdvanced", "description": "The advanced settings screen (developer mode, rageshake, push notification rules)"},
+        {"const": "SettingsHelp", "description": "The Help and About screen"},
+        {"const": "SettingsLegals", "description": "The settings screen with legals information"},
         {"const": "SettingsMentionsAndKeywords", "description": "The settings screen to manage notification mentions and keywords."},
         {"const": "SpaceMenu", "description": "The bottom sheet that list all space options"},
-        {"const": "SpaceMembers", "description": "Screen that displays the list of members of a space"}
+        {"const": "SpaceMembers", "description": "Screen that displays the list of members of a space"},
+        {"const": "RoomPreview", "description": "Screen that displays room preview if user hasn't joined yet"}
```

Comment:
Shouldn't we have 2 room preview screens? One for the room preview bottom sheet and one for the room preview screen?

Reply:
We don't need it right now i believe, probably it will be added later